docs/rules/await-fire-event.md

```diff
@@ -1,12 +1,14 @@
-# Enforce async fire event methods to be awaited (await-fire-event)
+# Enforce promises from fire event methods to be handled (await-fire-event)
 
-Ensure that promises returned by `fireEvent` methods are awaited
+Ensure that promises returned by `fireEvent` methods are handled
 properly.
 
 ## Rule Details
 
-This rule aims to prevent users from forgetting to await `fireEvent`
-methods when they are async.
+This rule aims to prevent users from forgetting to handle promise returned from `fireEvent`
+methods.
+
+> ⚠️ `fireEvent` methods are async only on `@testing-library/vue` package
```

Comment:
Looks like svelte is also wrapping fireEvent with a promise: https://github.com/testing-library/svelte-testing-library/blob/master/src/pure.js#L94-L106

Reply:
I see, I didn't know it. I'll update this note to clarify fire event methods are async on Vue and Svelte packages, but only supported official by the plugin for the former.